apps/web/src/pages/SkillFamilies/components/SkillFamilyTable.tsx

```diff
@@ -76,12 +75,15 @@ export const SkillFamilyTable = ({ skillFamilies }: SkillFamilyTableProps) => {
           description:
             "Title displayed for the Skill Family table Edit column.",
         }),
-        accessor: (sf) =>
+        id: "edit",
```

Comment:
Just figured out that nginx and asp.net are because it is sorting case-senstively.  I wonder if the accessor should run text through a normalize function to lowercase and remove diacritics for a primitive.

Reply:
asp.net and nginx is due to casing
upper case comes before lower case, hence why I liberally applied toLowerCase() in a bunch of places recently

Reply:
Fair enough.  I'll stub out an issue for that.

Reply:
I think **asp.net** and **nginx** are at the beginning (or end, if descending) because they begin with a lowercase letter. Same goes to why accented characters in French are treated differently.

One way to is to use localeCompare:

## Docs
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#using_options

## Example
https://github.com/GCTC-NTGC/TalentCloud/blob/17a4e0baa27dcd3f35f7d0958eb87202e256c277/resources/assets/js/components/JobBuilder/Skills/JobSkills.tsx#L275-L288

Reply:
#5655